connector.go

```diff
@@ -39,10 +38,8 @@ func (c *connector) Deploy(ctx context.Context, image string) (deployer.Plugin,
 		return nil, err
 	}
 
-	//nolint:godox
-	// TODO: Make this sleep workaround no longer needed.
 	// It's required to not crash podman
-	time.Sleep(500 * time.Millisecond)
+	time.Sleep(time.Duration(c.config.Deployment.StartDelayMS) * time.Millisecond)
```

Comment:
While waiting 0 milliseconds is fast, *not* waiting at all would be faster. It's not really a big deal, but I might be tempted to skip the call if the value is 0. 😁

Reply:
Updated to include an if statement. That'll save a few CPU cycles.